src/rtcmix/RTcmixMain.cpp

```diff
@@ -376,6 +388,7 @@ void
 RTcmixMain::run()
 {
    pthread_t   sockitThread;
+   pthread_t   OSC_ServerThread;
```

Comment:
We will never run sockit and OSC at the same time, so no need for two thread objects here.  Just create one called
"serverThread" and have both sets of code use it.